espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/ViewMatcherJson.kt

```diff
@@ -0,0 +1,51 @@
+package io.appium.espressoserver.lib.model
```

Comment:
this file almost fully copies `DataMatcherJsonDeserializer`. Can it be refactor to avoid the unnecessary duplication?